deploy/example_clusters/cr-cluster-spark221e2.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: "spark-instance"
 spec:
   app: spark221e2
+  serviceType: LoadBalancer
```

Comment:
I don't think we want to make this change; these "non-suffixed" YAML files are intended more for local k8s deployments that use NodePort.